auth_admin_passkey/models/res_config.py

```diff
@@ -14,8 +15,8 @@ class BaseConfigSettings(models.TransientModel):
     def get_default_auth_admin_passkey_send_to_admin(self, fields):
         return {
             'auth_admin_passkey_send_to_admin':
-            self.env["ir.config_parameter"].get_param(
-                "auth_admin_passkey.send_to_admin")
+            safe_eval(self.env["ir.config_parameter"].get_param(
+                "auth_admin_passkey.send_to_admin") or 'False')
```

Comment:
Let us start to use black code conventions for new code. Will ease migration to 13.0 and beyond. So quoting strings with double quotes.

Reply:
In order to make reviewing easier I've run 'black' before the two other commits, which are the functional fixes.